ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__installation-nightly-4.14.yaml

```diff
@@ -263,6 +263,22 @@ tests:
     test:
     - ref: cucushift-installer-check-cluster-health
     workflow: cucushift-installer-rehearse-gcp-ipi-xpn-cco-manual-users-static
+- as: vsphere-ipi-customized-resource-f28
+  cron: 56 22 18 * *
+  steps:
+    cluster_profile: vsphere-connected-2
+    env:
+      COMPUTE_NODE_CPU: "8"
```

Comment:
I am not sure the default cpu/mem/disk setting from installer, if possible, suggest to always set a different value with the default ones.